### ISSUE TITLE
Consider using a Fetch metadata header

### ISSUE THREAD

**Issue by letitz** [WICG/private-network-access#81]
We might want to let the server know that the request client is in a more public network, during the actual request, after the preflight request. This could take the shape of a fetch metadata header. Something like:

```
Sec-Fetch-Network: public
```

Modulo the appropriate amount of bikeshedding.

According to @mikewest in #80, some version of this was considered way back when CORS-RFC1918 was being initially designed.

**Comment by annevk** [comment]
If we do this we should probably only include it for non-public requests to avoid bloat in the common case.

**Comment by letitz** [comment]
Agreed! The default (not a private network request) can be implied by the absence of the header.
